sky/cli.py

```diff
@@ -2704,7 +2704,8 @@ def _down_or_stop_clusters(
 
     operation = 'Terminating' if down else 'Stopping'
     if idle_minutes_to_autostop is not None:
-        is_cancel = idle_minutes_to_autostop < 0
+        is_cancel = (idle_minutes_to_autostop ==
+                     constants.IDLE_MINUTES_TO_AUTOSTOP_FOR_CANCEL)
```

Comment:
Can we retain the `< 0` check instead of `== -1`? Some python users may already be relying of < 0 check and this change is not _required_ for this PR.